units/persistent_dns_assignment.go

```diff
@@ -4,20 +4,13 @@ import (
 	"context"
 	"fmt"
 
-	"github.com/evergreen-ci/evergreen"
-	"github.com/evergreen-ci/evergreen/cloud"
-	"github.com/evergreen-ci/evergreen/model/host"
 	"github.com/mongodb/amboy"
 	"github.com/mongodb/amboy/job"
 	"github.com/mongodb/amboy/registry"
-	"github.com/mongodb/grip"
-	"github.com/mongodb/grip/message"
-	"github.com/pkg/errors"
 )
 
 const (
-	persistentDNSAssignmentJobName   = "persistent-dns-assignment"
-	persistentDNSAssignmentBatchSize = 10
+	persistentDNSAssignmentJobName = "persistent-dns-assignment"
```

Comment:
random comment placement: 

General question: What is your plan for checking that any remaining persistent DNS jobs have finished draining out of the Amboy queue?

Reply:
I don't think it needs to be actually checked because it only enqueues one job per 15 mins, and there can only be one in the queue at once, so we just have to wait 15 mins after the deploy and the jobs will all be gone.

That being said, checking the [Amboy dashboard](https://mongodb.splunkcloud.com/en-US/app/search/evergreen_amboy?form.field1.earliest=-24h%40h&form.field1.latest=now&form.env=evergreen) will give info about the running jobs if needed.